src/interface/playerSelection/playerInfo/PlayerGearHeader.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PlayerGearHeader = ({ player, averageIlvl }) => {
+
+
+  return (
```

Comment:
```suggestion
const PlayerGearHeader = ({ player, averageIlvl }) => (
```

Reply:
Hadn't realised you could do this. All done